code/game/objects/structures/crates_lockers/closets/secure/hydroponics.dm

```diff
@@ -11,3 +11,90 @@
 	new /obj/item/cultivator(src)
 	new /obj/item/hatchet(src)
 	new /obj/item/secateurs(src)
+
+//Plocker
+/obj/structure/closet/secure_closet/plocker
+	name = "peach locker"
+	desc = "A huge hollow pitless peach."
+	locked = FALSE
+	icon_state = "plocker"
+	max_integrity = 40
+	armor = list(MELEE = 10, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, BIO = 0, RAD = 60, FIRE = 10, ACID = 0)
+	can_weld_shut = FALSE
+	damage_deflection = 0
+	material_drop_amount = 0
+	close_sound = "sound/misc/moist_impact.ogg"
+	open_sound = "sound/misc/soggy.ogg"
+	breakout_time = 100
+	var/isDegrading = FALSE
+	var/owner = null
+	var/lifespan = 1800
```

Comment:
while these vars youre just now making should

Reply:
Alright, I got rid of bunch of doc comments that weren't needed and added comments to new procs and variables only.